src/puppeteer.js

```diff
@@ -222,7 +224,7 @@ export const launchPuppeteer = (options = {}) => {
             sessionParamName: 'options.apifyProxySession',
         });
     }
-    if (!optsCopy.defaultViewport) {
+    if (optsCopy.defaultViewport === undefined) {
```

Comment:
Does it make sense to allow passing `"", 0, false, null` down to Puppeteer instead of just using the default when Puppeteer expects an `Object` anyway?

Reply:
For convenience I guess.